test/src/helpers.cc

```diff
@@ -1253,6 +1253,16 @@ void read_array(
   tiledb_query_free(&query);
 }
 
+int32_t num_commits(const std::string& array_name) {
+  Context ctx;
+  VFS vfs(ctx);
+
+  // Get all URIs in the array directory
```

Comment:
Let's filter out files that don't end with .wrt. As this is in test helpers, it might be used for other tests and it there are update, deletes, ignore or con files, this will return wrong information.